### ISSUE TITLE
dataserv-client 2.1.12 crashes

### ISSUE THREAD

**Issue by littleskunk** [Storj/pyp2p#16]
I decided to upgrade from dataserv-client 2.1.4 to the latest release (2.1.12). Though everything with 2.1.4 worked fine, 2.1.12 crashes immediately, even when checking the version.

I am running OSX 10.11.2, and have tried to uninstall/install/upgrade, all with the same result.

~~~~
% dataserv-client version
Traceback (most recent call last):
  File "/usr/local/bin/dataserv-client", line 6, in <module>
    from dataserv_client import cli
  File "/usr/local/lib/python2.7/site-packages/dataserv_client/cli.py", line 4, in <module>
    from dataserv_client import api
  File "/usr/local/lib/python2.7/site-packages/dataserv_client/api.py", line 7, in <module>
    import storjnode
  File "/usr/local/lib/python2.7/site-packages/storjnode/__init__.py", line 3, in <module>
    from . import network  # NOQA
  File "/usr/local/lib/python2.7/site-packages/storjnode/network/__init__.py", line 1, in <module>
    from . import node  # NOQA
  File "/usr/local/lib/python2.7/site-packages/storjnode/network/node.py", line 13, in <module>
    from pyp2p.unl import UNL
  File "/usr/local/lib/python2.7/site-packages/pyp2p/unl.py", line 6, in <module>
    from .lib import *  # FIXME * is evil!
  File "/usr/local/lib/python2.7/site-packages/pyp2p/lib.py", line 37, in <module>
    ip = IPDB()
  File "/usr/local/lib/python2.7/site-packages/pyroute2/__init__.py", line 89, in __new__
    cls.register(cls.registry['class'])
  File "/usr/local/lib/python2.7/site-packages/pyroute2/__init__.py", line 69, in __getitem__
    [name], 0)
  File "/usr/local/lib/python2.7/site-packages/pyroute2/ipdb/__init__.py", line 396, in <module>
    from pyroute2.iproute import IPRoute
  File "/usr/local/lib/python2.7/site-packages/pyroute2/iproute.py", line 109, in <module>
    from pyroute2.netlink.rtnl.tcmsg import get_htb_parameters
  File "/usr/local/lib/python2.7/site-packages/pyroute2/netlink/rtnl/tcmsg.py", line 29, in <module>
    _psched = open('/proc/net/psched', 'r')
IOError: [Errno 2] No such file or directory: '/proc/net/psched'
~~~~

**Comment by robertsdotpm** [comment]
Thanks. I don't think ipdb works at all on OSX as its meant for Linux only. Will make the appropriate changes tomorrow

**Comment by littleskunk** [comment]
Same problem with FreeBSD
catbref [15:15] 
File "/usr/local/lib/python2.7/site-packages/pyroute2/netlink/rtnl/tcmsg.py", line 29, in <module>
   _psched = open('/proc/net/psched', 'r')
IOError: [Errno 2] No such file or directory: '/proc/net/psched'

​[15:16] 
so basically same issue

Possible solution
f483 We should report it ipstream if not already done

**Comment by littleskunk** [comment]
I have the same problem with my debian nas.

**Comment by littleskunk** [comment]
version 0.5.8 is working. 0.5.9-0.6.1 are not working.

```
pip install pyp2p==0.5.8
```

**Comment by littleskunk** [comment]
Fixed with pyp2p 0.6.4. Retest with my debian nas passed.
